### DISCUSSION TITLE
Problem connecting to Tinyg v7

### DISCUSSION THREAD

**Discussion by Dougremo** [winder/Universal-G-Code-Sender#2268] [Help]
I'm having an issue connecting a TinyG-23 v7 to UGS.

Below is the build info for this board. I believe it was from a Shapeoko that I purchased 10 or so years ago.
#### TinyG version 0.94 (build 339.09) "GoGo Boots" ####

I can connect to the board and jog with Coolterm terminal window, but the UGS software just says connecting and gives an unknown board message in the command window.

I installed the latest version of UGS on my Dell Inpiiron Laptop with the java included.

Any ideas would be great!

Doug

**Comment by breiler** [comment]
I know there were quite large changes in the protocol between version 0.96 and 0.97. It is likely that there are other significant protocol changes since version 0.94 which won't work with UGS.

I just booted up their latest version 0.97 (build 440.20) and it seems to run ok in UGS 2.0.18.

**Comment by Dougremo** [comment]
So just a simple update of the firmware will fix this issue?Sent from Samsung Galaxy smartphone.
-------- Original message --------From: Joacim Breiler ***@***.***> Date: 7/28/23  2:32 PM  (GMT-06:00) To: winder/Universal-G-Code-Sender ***@***.***> Cc: Dougremo ***@***.***>, Author ***@***.***> Subject: Re: [winder/Universal-G-Code-Sender] Problem connecting to Tinyg v7 (Discussion #2268) 
I know there were quite large changes in the protocol between version 0.96 and 0.97. It is likely that there are other significant protocol changes since version 0.94 which won't work with UGS.
I just booted up their latest version 0.97 (build 440.20) and it seems to run ok in UGS 2.0.18.

**Comment by breiler** [comment]
I can't guarantee anything, but it connects ok for me.
